include/ginkgo/core/base/array.hpp

```diff
@@ -93,6 +93,14 @@ class const_array_view {
      *
      * @return the number of elements in the array view
      */
+    size_type get_size() const noexcept { return num_elems_; }
```

Comment:
Maybe for uniformity and clarity, this internal member should also be renamed.
```suggestion
    size_type get_size() const noexcept { return size_; }
```